editor/components/TableWrapper.tsx

```diff
@@ -84,9 +82,30 @@ export default function TableWrapper<T extends Record<string, unknown>>({
     setColumnVisibilitySettings,
   } = useVisibleColumns(columns);
 
+  /**
+   * Array of columns which should be fetched - this is
+   * is the combination of visibile columns + fetchAlways columns
+   */
```

Comment:
this new `fetchAlways` setting came out of my map work, specifically to handle when we want to include `latitude` and `longitude` columns in our query but we don't need to show them in the table or expose them in the col picker.